app/apptesting/gamm.go

```diff
@@ -16,6 +16,8 @@ var DefaultAcctFunds sdk.Coins = sdk.NewCoins(
 	sdk.NewCoin("foo", sdk.NewInt(10000000)),
 	sdk.NewCoin("bar", sdk.NewInt(10000000)),
 	sdk.NewCoin("baz", sdk.NewInt(10000000)),
+	sdk.NewCoin("eth", sdk.NewInt(10000000)),
```

Comment:
Nit: consider using ETH and USDC to consolidate with other testing denoms

Reply:
Removed, was there for early testing